MovingBlocks/TerasologyLauncher
# Set column width in logging table view

Fixes the column width for the first two columns (timestamp and log
level), and uses a resize policy which will use the remaining space for
the third column (log message).

Fixes #466

### How to test

Check this branch out and start the launcher. The **Log** tab should show a table with properly sized columns, like so:

![image](https://user-images.githubusercontent.com/1448874/68549725-af3e6d00-03fb-11ea-85ad-3486a74c4c20.png)
